eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -285,6 +285,23 @@
           "code": "java.annotation.added",
           "new": "class com.azure.cosmos.models.ChangeFeedProcessorItem",
           "justification": "Modifies the type of changeFeedMetaData from ChangeFeedMetaData to JsonNode."
+        },
+        {
+          "code": "java.method.returnTypeChanged",
+          "old": "method com.azure.resourcemanager.compute.models.StorageAccountType com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation::storageAccountType()",
+          "new": "method com.azure.resourcemanager.compute.models.EdgeZoneStorageAccountType com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation::storageAccountType()",
+          "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+        },
+        {
+          "code": "java.method.parameterTypeChanged",
+          "old": "parameter com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation::withStorageAccountType(===com.azure.resourcemanager.compute.models.StorageAccountType===)",
+          "new": "parameter com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation::withStorageAccountType(===com.azure.resourcemanager.compute.models.EdgeZoneStorageAccountType===)",
+          "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+        },
+        {
+          "code": "java.field.removed",
+          "old": "field com.azure.resourcemanager.compute.models.StorageAccountType.STANDARD_SSD_LRS",
+          "justification": "Fix field removed."
         }
```

Comment:
We need to know the reason from Compute RP, especially the removal of `STANDARD_SSD_LRS`.

For model rename, we may use `--rename-model` to rename it back.